addons/pos_sale/models/sale_order.py

```diff
@@ -59,6 +59,16 @@ def _compute_amount_to_invoice(self):
             order_amount = sum(order.sudo().pos_order_line_ids.filtered(lambda pol: pol.sale_order_line_id.is_downpayment).mapped('price_subtotal_incl'))
             order.amount_to_invoice -= order_amount
 
+    @api.depends('order_line.pos_order_line_ids')
+    def _compute_amount_invoiced(self):
+        super()._compute_amount_invoiced()
+        for order in self:
+            if order.invoice_status == 'invoiced':
+                continue
+            # We need to account for the downpayment paid in POS with and without invoice
+            order_amount = sum(order.sudo().pos_order_line_ids.filtered(lambda pol: pol.order_id.state in ['paid', 'done'] and pol.sale_order_line_id.is_downpayment).mapped('price_subtotal_incl'))
```

Comment:
```suggestion
            order_amount = sum(order.sudo().pos_order_line_ids.filtered(lambda pol: pol.order_id.state in ['paid', 'done', 'invoiced'] and pol.sale_order_line_id.is_downpayment).mapped('price_subtotal_incl'))
```

Reply:
As mentioned in the comment above the line, you should also take into account the pos.order.line from an invoiced pos.order.